apps/public-reference/pages/applications/review/summary.tsx

```diff
@@ -39,29 +94,168 @@ export default () => {
       </FormCard>
 
       <FormCard>
-        <p className="text-bold">
+        <p className="form-card__back">
           <strong>
             <Link href="/applications/review/demographics">Back</Link>
           </strong>
         </p>
 
-        <h2 className="form-card__title is-borderless">Summary</h2>
-
-        <hr />
-
-        <form className="mt-10" onSubmit={handleSubmit(onSubmit)}>
-          (FORM)
-          <div className="text-center mt-6">
-            <Button
-              filled={true}
-              onClick={() => {
-                //
-              }}
-            >
-              Next
-            </Button>
+        <div className="form-card__lead">
+          <h2 className="form-card__title is-borderless">
+            Take a moment to review your information before submitting your application.
+          </h2>
+        </div>
+
+        <h3 className="px-8 py-4 bg-gray-200">
+          You
+          <EditLink href="/applications/contact/name" />
+        </h3>
+
+        <div className="form-card__group mx-0">
+          <ReviewItem label={"Name"}>
+            {application.applicant.firstName} {application.applicant.middleName}{" "}
+            {application.applicant.lastName}
```

Comment:
Are we guaranteed to have all 3 values here? (e.g. is `middleName` optional) If not, we'll have extra spaces with this, I think I'd probably do:

```tsx
{["firstName", "middleName", "lastName"].map(key => application.applicant[key]).filter(value => value && value.length > 0).join(" ")}
```

Reply:
Good thinking…I believe HTML will swallow any extra whitespace so might not need exact accuracy in this context.

Reply:
Ya, I guess that's probably true. Still feels wrong though ;-)